inductiva/_benchmarks/benchmark.py

```diff
@@ -0,0 +1,119 @@
+"""API for Benchmarking"""
+import json
+from typing import Self
+from inductiva import types
+from inductiva.simulators.simulator import Simulator
+from inductiva.projects.project import Project
+from inductiva.resources import machine_groups
+
+
+class Benchmark(Project):
+    """Represents the benchmark runner."""
+
+    def __init__(self, name: str, append: bool = True):
```

Comment:
Should the default value for `append` be set to `True`?
For projects we are using append False by default to make the action of adding things to a project be explicit so the user does not add anything by mistake.

Reply:
In this case, I think it should. What should happen if a user runs the same benchmark file twice? Currently, it includes additional runs to the benchmark (e.g., makes the results / performance metrics more reliable). This behavior is due to ```append``` being set to ```True``` by default.